scripts/onload.js

```diff
@@ -33,24 +33,33 @@ map.on('click', 'school_house_senate_districts_UNION-poly', function (e) {
 	popupYear = 0;
 });
 
-
-
+// skip the landing page if we already have ?=arguments
+if (urlParams["districts"]) {
+	document.getElementById('landing').style.visibility = "hidden";
+}
```

Comment:
This code makes the landing page go away if the URL has `?district=` appended to it.  That seems like good UI to me mainly because when you switch between house, senate and school districts the whole page gets reloaded, so we need this to not make people click through the landing page over and over again.  But if we want to take it out, it's just this `if` statement.

Reply:
This seems like a good idea. Testing the latest just now, the landing page does blink into existence momentarily before vanishing when switching between ISDs, House, and Senate districts. Probably not a huge deal but I imagine RYHT will ask us to mute that behavior. Also nice choice on the Hat & Boots image for a placeholder ;) I will ask RYHT what they would like to use instead. The other potential issue is they might request stats box functionality for the ISDs, to match the House and Senate behavior. How much of a lift is that for you?

Reply:
@eldang one more question:  can I work on the formatting in this branch without running you over?

Reply:
@coregis I think what we should do is merge this PR to `school-districts-as-background-layer` even though there will be more changes to come in here.  Then you can work in that branch and I can continue in this one, with a little more safety from conflicting edits.

If I can filter the points, then I can activate the stats box too - the one thing is contingent on the other, but once filtering is in place the remainder is easy.  If I can leave the spatial join to you, I'll try to figure out smoother hiding of the landing page meanwhile.

Reply:
@eldang ha ha I see now you already laid all this out for me on Friday, sorry for missing that :)

I will work on the spatial join for all the programs so they are filterable by ISD. I'm not sure I understand your suggestion with this PR though--are you saying we merge this PR (#16) into ryht-feedback-layout-changes, then I do formatting/layout stuff there, and you keep working on functionality in school-districts-as-background-layer?

Reply:
@coregis Yeah, because then when we do come to merge your and my parallel edits later there will be fewer changes to deal with.